README.md

```diff
@@ -2,6 +2,8 @@
 
 [Eppo](https://www.geteppo.com/) is a modular flagging and experimentation analysis tool. Eppo's .NET SDK is built to make assignments in multi-user server side contexts, compatible with Dot Net 7.0 Runtime. Before proceeding you'll need an Eppo account.
 
+Despite the name, this SDK runs in both server and client applications.
```

Comment:
Should we update the repo name? (Or if that's a bad idea, create a new repo?)

Reply:
I agree, we should at least update the repo name. I believe that we can do this without breaking the world as there should be finite few references to this repo name, and Github has Repository Redirecting set up for renamed repos to allow the repo to be located at the previous location. @leoromanovsky what are your thoughts?

Reply:
We should probably give all our repo names a combined once-over, and do this as part of that